README.md

```diff
@@ -866,6 +867,18 @@ This is a temporary state and once a stable is released it is changed back to th
 ```
 
 
+## Integrity Tests
+
+A [number of integrity tests](tests/IntegrityTests) validate that samples and snippets conform to conventions that make the documentation easier to maintain. PRs that fail these tests will not be mergeable.
+
+The integrity tests include:
+
+* [Samples should not be multi-targted](tests/IntegrityTests/ProjectFrameworks.cs) using the `<TargetFrameworks>` (plural) element, and instead have separate solutions and projects to support .NET Framework and .NET Core, all using the same underlying code, i.e.:
```

Comment:
I think it'd be useful here to explain why we use this approach, as this may change in the future with updated tooling support and it'll be easier to catch that rules are outdated.